pkg/ccl/changefeedccl/kvfeed/kv_feed.go

```diff
@@ -115,10 +111,8 @@ func Run(ctx context.Context, cfg Config) error {
 	g := ctxgroup.WithContext(ctx)
 	g.GoCtx(cfg.SchemaFeed.Run)
 	g.GoCtx(f.run)
-	if cfg.FeedWatcher != nil {
-		g.GoCtx(cfg.FeedWatcher)
-	}
 	err := g.Wait()
+	log.Errorf(ctx, "kvfeed OVERALL err %v\n", err)
```

Comment:
I don't think we should be printing this error, particularly all caps.
Seeing error is going to be normal; it will happen every time some transient error happens, etc and this error will be too ominous.
If you insist, emit it only if e.g. `if log.V(1)` or something like that.

Reply:
Sorry I meant to delete it. I honestly don't know how I deleted the `defer` but not the log line.